src/BullOak.Repositories/Session/IValidationError.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace BullOak.Repositories.Session
+{
+    using System;
+
+    public interface IValidationError
+    {
+        string Message { get; }
+        Exception GetAsException();
```

Comment:
`GetAsException` could be an extension method to keep this interface cleaner.

Reply:
Hey @iblazhko, I want it here so that you can provide a custom validator which throws custom exceptions